### PR TITLE
Release of 0.3.7

### PR DESCRIPTION
**Pull request by mmcky**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -53,6 +53,7 @@ install:
     # Add path, activate `conda` and update conda.
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
     - cmd: conda update --yes --quiet conda
+    - cmd: conda clean --all
```
Comment (jakirkham) [inline appveyor.yml]: Please do not directly modify any files outside of the `recipe` directory. If you are concerned about an issue related to them, please try [re-rendering]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ). If that doesn't work, please ping core. Thanks.

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

File: appveyor.yml (hunk 1 above)
Comment (mmcky) [reply appveyor.yml]: OK sure. Thanks for the tag. I will undo this change now.

Comment (jakirkham) [reply appveyor.yml]: I think the re-rendering fixed it. So shouldn't need further adjustment.

Comment (mmcky) [reply appveyor.yml]: thanks for the help @jakirkham.

**Pull request merged**